boltdesignsystem/bolt
# DS-464 add bell and flag icons

## Jira

https://pegadigitalit.atlassian.net/browse/DS-464

## Summary

Adds `bell-solid` and `flag-solid` icons.

## Details

1. Compressed and cleaned up SVGs
2. Imported the SVGs to icons

## How to test

Run the branch locally and check the icon name variations doc, make sure the `bell-solid` and `flag-solid` icons are displaying correctly in both light and dark themes.
